src/plugins/core/header_size.ts

```diff
@@ -1,48 +1,25 @@
-import { DEFAULT_CELL_HEIGHT, DEFAULT_CELL_WIDTH } from "../../constants";
-import { deepCopy, getAddHeaderStartIndex, getDefaultCellHeight, lazy, range } from "../../helpers";
+import { DEFAULT_CELL_WIDTH } from "../../constants";
+import { deepCopy, getAddHeaderStartIndex, range, removeIndexesFromArray } from "../../helpers";
 import { Command, ExcelWorkbookData, WorkbookData } from "../../types";
-import { CellPosition, Dimension, HeaderIndex, Lazy, Pixel, UID } from "../../types/misc";
+import { Dimension, HeaderIndex, Pixel, UID } from "../../types/misc";
 import { CorePlugin } from "../core_plugin";
-
-interface HeaderSize {
-  manualSize: Pixel | undefined;
-  computedSize: Lazy<Pixel>;
-}
+import { DEFAULT_CELL_HEIGHT } from "./../../constants";
 
 interface HeaderSizeState {
-  sizes: Record<UID, Record<Dimension, Array<HeaderSize>>>;
+  sizes: Record<UID, Record<Dimension, Array<Pixel | undefined>>>;
 }
-
 export class HeaderSizePlugin extends CorePlugin<HeaderSizeState> implements HeaderSizeState {
-  static getters = ["getRowSize", "getColSize"] as const;
+  static getters = ["getUserRowSize", "getColSize"] as const;
 
-  readonly sizes: Record<UID, Record<Dimension, Array<HeaderSize>>> = {};
+  readonly sizes: Record<UID, Record<Dimension, Array<Pixel | undefined>>> = {};
```

Comment:
Beacause in core, row size need to be undefined. We cannot set undefined row sized to DEFAULT_CELL_HEIGHT, because the UI plugin need to make the difference between rows where the user manually set a size, and the one he didn't.